apps/web/src/components/shared/ProfileImage.styles.ts

```diff
@@ -1,7 +1,7 @@
 import { css } from '../../styled-system/css';
 
 export const previewContainerStyles = css({
-  bg: 'surface.page',
+  bgColor: 'surface.popover',
```

Comment:
found this color to be closest to the one of designs, but is incorrect for light theme